app/javascript/dashboard/store/modules/helpCenterArticles/actions.js

```diff
@@ -0,0 +1,24 @@
+import helpCenterArticlesAPI from 'widget/api/conversationPublic';
+
+export const actions = {
+  fetchAllArticles: async ({ commit }) => {
+    try {
+      commit('setUIFlag', { isFetching: true });
+      const { data } = await helpCenterArticlesAPI.get();
+      data.forEach(helpCenter => {
+        const { id: helpCenterId } = helpCenter;
+
+        commit('addHelpCenterArticleEntry', helpCenter);
+        commit('addHelpCenterArticleId', helpCenterId);
+        commit('setHelpCenterArticleUIFlag', {
```

Comment:
I know it's in the draft stage. I would suggest using the constants file for constants used.